exs/mundo_1/javascript/010.js

```diff
@@ -0,0 +1,16 @@
+/*
+Desafio 010
+
+Problema: Crie um programa que leia quanto dinheiro uma pessoa tem na carteira e mostre quantos dólares ela pode comprar
+
+Resolução do problema:
+*/
+
+const input = require('readline-sync').question
+
+let qntDinheiro = Number(input("Quanto dinheiro você tem? R$"))
+let valorDolar = 7.85 // penando no futuro
```

Comment:
Remova o comentário desnecessário
```suggestion
let valorDolar = 7.85
```